skill-tree /src/test/java/com/RDS/skilltree/TestContainerManger.java

```diff
@@ -0,0 +1,24 @@
+package com.RDS.skilltree;
+
+import org.springframework.boot.testcontainers.service.connection.ServiceConnection;
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.springframework.test.context.DynamicPropertySource;
+import org.testcontainers.containers.MySQLContainer;
+
+abstract class TestContainerManger {
+    @ServiceConnection
+    static final MySQLContainer mysqlContainer;
+
+    static {
+        mysqlContainer = new MySQLContainer("mysql:latest")
```

Comment:
Use the mysql image version as the one that is actually used in the system.

Reply:
Using mysql : 8.1 as in the application as well we didn't defined any specific image earlier, so mentioned 8.1 there too.